shadowtools/src/shadowtools/shadow_exec.py

```diff
@@ -50,6 +51,7 @@ def _main(
     stderr: TextIO = sys.stderr,
     shadow_bin: Path = Path("shadow"),
     model_unblocked_syscall_latency: bool = True,
+    shadow_args: List[str] = [],
```

Comment:
I think this would be better as `= None` and then `if shadow_args is None: shadow_args = []`. Especially since we append to this global list below (`shadow_args.append(...)`). I know `_main` should only be called once per program, but I think it's best to avoid this footgun anyways.

Reply:
(since mutating a list that the caller passes is also potentially surprising)